Update README.md

wise4rmgodroot opened the pull request:
I updated the readme to included more details that will help external developers get started easily.

alepc253 commented:
Great! I would place that functionality in a different place. Maybe a different bullet will do (I mean, to set a difference to "Quick start", "The stack", etc)

bguiz commented:
> Great! I would place that functionality in a different place. Maybe a different bullet will do (I mean, to set a difference to "Quick start", "The stack", etc)

@alepc253 I have removed the link to search for the `<ul>`  and placed it just below, and made it bold, which I think should satisfy your comment. Going to merge this now to get the functionality. Let's iterate again if we have more ideas for the design or placement.

wise4rmgodroot committed:
Update README.md

I updated the readme to included more details that will help external developers get started easily.
